Sources/Authentication/Helpers/NetworkHelper.swift

```diff
@@ -39,15 +39,17 @@ extension AuthenticationHandler {
 
 extension AuthenticationHandler {
     func createAuthorizationURL() -> URL? {
+        guard let configuration = configuration else { return nil }
```

Comment:
You could throw .invalidConfiguration error here if the self.configuration is nil instead of returning nil, maybe that would be more readable. If you like to return nil instead maybe make a comment that nil is returned when an error happens

Reply:
I'm throwing an error when I use this function

```
guard let url = createAuthorizationURL() else { return continuation.resume(returning: .failure(CustomError.invalidURL)) }
```

Reply:
Yes i saw that, I was kinda confused why createAuthorizationURL() returned an optional, but could read in the code nil would be returned if the function failed. Im fine with it

Reply:
It's not a public function and internally you would understand it but next time I'll try be better